README.md

```diff
@@ -1,3 +1,60 @@
 # NVA Data Report API
 
-This repository contains the NVA data report API.
\ No newline at end of file
+This repository contains the NVA data report API.
+
+## How to run a bulk upload
```

Comment:
I'd include a short description of the steps as a simple overview, something like:

The steps below can be outlined briefly as:
- Pre-run
  - stop incoming live-update events
  - delete data from previous runs
  - delete all data in database
- Bulk upload
  - Generate batches of document keys for upload
  - Transform the data to a format compatible with the bulk-upload action
  - Initiate bulk upload
  - Verify data integrity
- Post-run
  - start incoming live-update events

Reply:
Additionally, I'd probably describe the use of the SageMaker notebook in a footnote in order to provide additional information on how to actually query (including the dreaded expired presigned URI issue)